api-service/database/licenses.go

```diff
@@ -17,100 +17,114 @@ package database
+								"count": mu.APOMax("$$value.count", "$$this.count"),
+								"dbs": bson.M{
+									"$concatArrays": bson.A{
+										"$$value.dbs",
+										bson.A{"$$this.name"},
 									},
-									"$$val.count",
-								),
-							}),
-							"db",
-							mu.APOGreater("$$db.count", 0),
-						),
-						bson.M{"count": 0, "dbs": bson.A{}},
-						bson.M{
-							"count": mu.APOMax("$$value.count", "$$this.count"),
-							"dbs": bson.M{
-								"$concatArrays": bson.A{
-									"$$value.dbs",
-									bson.A{"$$this.name"},
 								},
 							},
+						),
+					}),
+					mu.APMatch(bson.M{
+						"databases.count": bson.M{
+							"$gt": 0,
 						},
-					),
-				}),
-				mu.APMatch(bson.M{
-					"databases.count": bson.M{
-						"$gt": 0,
-					},
-				}),
-				AddAssociatedClusterNameAndVirtualizationNode(olderThan),
-				mu.APGroup(mu.BsonOptionalExtension(full, bson.M{
-					"_id": mu.APOCond(
-						"$clusterName",
-						mu.APOConcat("cluster_§$#$§_", "$clusterName"),
-						mu.APOConcat("hostname_§$#$§_", "$hostname"),
-					),
-					"license":    mu.APOMaxAggr("$databases.count"),
-					"clusterCpu": mu.APOMaxAggr("$clusterCpu"),
-				}, bson.M{
-					"hosts": mu.APOPush(bson.M{
-						"hostname":  "$hostname",
-						"databases": "$databases.dbs",
 					}),
-				})),
-				mu.APSet(bson.M{
-					"license": mu.APOCond(
-						"$clusterCpu",
-						mu.APODivide("$clusterCpu", 2),
-						"$license",
-					),
-				}),
-				mu.APGroup(mu.BsonOptionalExtension(full, bson.M{
-					"_id":   0,
-					"value": mu.APOSum("$license"),
-				}, bson.M{
-					"hosts": mu.APOPush("$hosts"),
-				})),
-				mu.APOptionalStage(full, mu.MAPipeline(
-					mu.APUnwind("$hosts"),
-					mu.APUnwind("$hosts"),
-					mu.APGroup(bson.M{
+					AddAssociatedClusterNameAndVirtualizationNode(olderThan),
+					mu.APGroup(mu.BsonOptionalExtension(isFull, bson.M{
+						"_id": mu.APOCond(
+							"$clusterName",
+							mu.APOConcat("cluster_§$#$§_", "$clusterName"),
+							mu.APOConcat("hostname_§$#$§_", "$hostname"),
+						),
+						"license":    mu.APOMaxAggr("$databases.count"),
+						"clusterCpu": mu.APOMaxAggr("$clusterCpu"),
+					}, bson.M{
+						"hosts": mu.APOPush(bson.M{
+							"hostname":  "$hostname",
+							"databases": "$databases.dbs",
+						}),
+					})),
+					mu.APSet(bson.M{
+						"license": mu.APOCond(
+							"$clusterCpu",
+							mu.APODivide("$clusterCpu", 2),
+							"$license",
+						),
+					}),
+					mu.APGroup(mu.BsonOptionalExtension(isFull, bson.M{
 						"_id":   0,
-						"value": mu.APOMaxAggr("$value"),
+						"value": mu.APOSum("$license"),
+					}, bson.M{
 						"hosts": mu.APOPush("$hosts"),
-					}),
+					})),
+					mu.APOptionalStage(isFull, mu.MAPipeline(
+						mu.APUnwind("$hosts"),
+						mu.APUnwind("$hosts"),
+						mu.APGroup(bson.M{
+							"_id":   0,
+							"value": mu.APOMaxAggr("$value"),
+							"hosts": mu.APOPush("$hosts"),
+						}),
+					)),
 				)),
-			)),
 			mu.APSet(bson.M{
```

Comment:
Where are the `)),` characters aligned to `mu.APSet(bson.M{`?

Reply:
I just changed indentation: in fact, there are still two `))` between mu.APOptionalStage and mu.APSet